### ISSUE TITLE
Random respring when attempt to open app switcher

### ISSUE THREAD

**Issue by ganjarpanji** [pwn20wndstuff/Undecimus#2058]
**Describe the bug**
Previous version of unc0ver(4.x.x) give me a random respring a lot, but with 5.0.1 version I experienced better, the random respring is reduced significantly.

but one thing that certain the respring always occurs when I tried to open app switcher. probably this could help devs to find out what causes it.

**To Reproduce**
1. Open app switcher (the respring randomly occurs)

**Screenshots**

**Device (please complete the following information):**
 - iOS Version: iOS 13.5
 - iOS Device: A12 (iPhone XS)
 - unc0ver Version: 5.0.1

**Place an "x" between the brackets if true:**
 - [X] this is a bug others will be able to reproduce
 - [ ] this issue is present with all tweaks uninstalled(except for default packages) or disabled
 - [ ] this issue is present after a rootfs restore
 - [X] this issue is present on the latest version of unc0ver

**Logs**
If applicable, add logs or error messages here.

**Comment by LoveJessyChen** [comment]
Me Too.I use unc0ver 5.0.1 on iPhone X with iOS 13.5

**Comment by Hamlock-maneuverr** [comment]
Are you using the tweak "gesto"  or something similar? If so uninstall and see if that fixes it.

**Comment by ganjarpanji** [comment]
nope, probably tweak related to app switcher I'm using QuitAll. I'll try to disable this tweak and let you know if that solves the problem.

**Comment by LoveJessyChen** [comment]
Nope，I did not use “ gesto” tweak.

**Comment by ganjarpanji** [comment]
I could confirm, disabling QuitAll is not solve this.

**Comment by Hamlock-maneuverr** [comment]
Then you need to go through each and every tweak until you find the one causing the problem.

**Comment by ganjarpanji** [comment]
Since previous version the random respring is oftenly happen, i assume there is no problem with my installed tweaks.
